label-nodes.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 this=`basename $0`
-if [ $# -ne 1 ]; then
+if [ $# -gt 1 ]; then
     echo Usage: $this IMAGE[:TAG]
```

Comment:
You need to update the usage message

Reply:
Ha, thanks for pointing this out! Fixed now